10월/4주차/20251020/leejaeyeon.py

```
@@ -0,0 +1,54 @@
+
+
+pos = [(0,1),(1,0),(0,-1),(-1,0)]
+
+N = int(input())
+
+K = int(input())
+
+matrix = [[0]*N for _ in range(N)]
+matrix[0][0] = 2 # 뱀 머리
+
+for i in range(K):
```

Ruff (0.14.1)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)